### PR TITLE
Playwright

### PR DESCRIPTION
**Pull request by rimi-itk**
#### Description

Adds [Playwright](https://playwright.dev/) tests to the mix.

> [!NOTE]  
> This pull request builds on https://github.com/itk-dev/ai-screening/pull/36 to make tests succeed.

#### Checklist

- [x] My code is covered by test cases.
- [x] My code passes our test (all our tests).
- [x] My code passes our static analysis suite.
- [x] My code passes our continuous integration process.

If your code does not pass all the requirements on the checklist you have to add a comment explaining why this change 
should be exempt from the list.

#### Additional comments or questions

If you have any further comments or questions for the reviewer please add them here.

### CODE REVIEW

File: docs/Testing.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+create-playwright@1.17.134
+Ok to proceed? (y)
+
+
+> app@1.0.0 npx
+> create-playwright
+
+Getting started with writing end-to-end tests with Playwright:
+Initializing project in '.'
+✔ Do you want to use TypeScript or JavaScript? · TypeScript
```
Comment (sinejespersen) [inline docs/Testing.md:51]: TypeScript ?

**Review by sinejespersen — APPROVED** [review]
Why TypeScript?

File: Taskfile.yml [hunk 2]
```diff
@@ -136,8 +136,49 @@ tasks:
...
+      - task drush -- cache:rebuild
+      - task compose -- run --rm node npm install
+      - task compose -- run --rm playwright npx playwright install
+
   set-user-roles:
     desc: "Set some user's roles"
     cmds:
       - task drush -- user:role:add 'administrator' admin
     silent: true
+
```
Comment (sinejespersen) [inline Taskfile.yml:179]: Prettier in readme?

**Review by sinejespersen — CHANGES_REQUESTED** [review]

File: Taskfile.yml (hunk 2 above)
Comment (rimi-itk) [reply Taskfile.yml:179]: Added in https://github.com/itk-dev/ai-screening/pull/39/commits/8628150c6237799cefb7b70a3d204ac8d4a0fe86.

**Review by sinejespersen — APPROVED** [review]

**Pull request merged**